Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv

```diff
@@ -0,0 +1,123 @@
+﻿Result
```

Comment:
these are not data test result

Reply:
having results

Reply:
Did you use correct Schema name? All fields are currently showing unnormalized

Reply:
no it's not showing

Reply:
rolled back changes